pandas/tests/frame/test_missing.py

```diff
@@ -814,6 +814,18 @@ def test_interp_raise_on_only_mixed(self):
         with pytest.raises(TypeError):
             df.interpolate(axis=1)
 
+    def test_interp_raise_on_all_object_dtype(self):
+        df = DataFrame({
```

Comment:
Can you add the GitHub issue number as a comment?  See two tests below for an example.  It's useful if some makes changes that cause this test to break, as it allows them to quickly find the discussion and context for the test.